kura/org.eclipse.kura.core.log/pom.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
 
-    Copyright (c) 2011, 2014 Eurotech and/or its affiliates
+    Copyright (c) 2011, 2014 Eurotech and others
```

Comment:
@dwoodard1 Could you confirm which identifier you want as legal entity for Eurotech: "Eurotech" or "Eurotech and/or its affiliates".